src/components/Restoration/ApprovalType.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <div id="approval-type">
+    <div class="section-container" :class="{ 'invalid-section': invalidSection }">
+      <ApprovalTypeShared
```

Comment:
Does the label change to red when the approval type is not valid?

Reply:
This code changes the left border of the div to be red.

There is other code that changes the label to red.

![image](https://user-images.githubusercontent.com/10002889/221000789-84f7e001-6d7a-421f-9513-9bc46c2afd4e.png)

Reply:
@JazzarKarim OK to resolve this conversation?